scipy/special/tests/test_support_alternative_backends.py

```diff
@@ -89,4 +89,5 @@ def test_support_alternative_backends(xp, f_name_n_args, dtype, shapes):
     ref = xp.asarray(f(*args_np), dtype=dtype_xp)
 
     eps = np.finfo(dtype_np).eps
-    xp_assert_close(res, ref, atol=10*eps, check_0d=False)
+    ref = ref[()] if ref.ndim == 0 else ref
+    xp_assert_close(res, ref, atol=10*eps)
```

Comment:
sometimes this is nonzero-dimensional, so array-api-strict isn't happy with the 0d indexing. That's fine, guard on `ndim`.